types/svg2ttf/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for svg2ttf 5.0
+// Project: https://github.com/fontello/svg2ttf
+// Definitions by: Ewan Morrison <https://github.com/ewan-m>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function svg2ttf(svgFontString: string, options?: FontOptions): MicroBuffer;
```

Comment:
```suggestion
/**
 * Converts SVG fonts to TTF format
 */
declare function svg2ttf(svgFontString: string, options?: FontOptions): MicroBuffer;
```
This will be visilbe as the module description and as the funtion documentation